### PR TITLE
Hide resources editor if no course version is supplied

### PR DESCRIPTION
**Pull request by bethanyaconnor**
First and simplest step for [PLAT-505](https://codedotorg.atlassian.net/browse/PLAT-505): hide the resource editor if no course version is provided.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx [hunk 1]
```diff
@@ -301,13 +301,15 @@ class LessonEditor extends Component {
...
         </CollapsibleEditorSection>
 
-        <CollapsibleEditorSection
-          title="Resources"
-          collapsed={true}
-          fullWidth={true}
-        >
-          <ResourcesEditor courseVersionId={this.props.courseVersionId} />
-        </CollapsibleEditorSection>
+        {this.props.courseVersionId && (
```
Comment (dmcavoy) [inline apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx]: Should there be some kind of message as to why it is hidden?

**Comment by bethanyaconnor** [comment]
@dmcavoy good thought! How's this?
Screenshot 2020-12-08 at 7 27 43 AM

Definitely open for feedback on wording here.

**Comment by davidsbailey** [comment]
> @dmcavoy good thought! How's this?
> Screenshot 2020-12-08 at 7 27 43 AM
> 
> Definitely open for feedback on wording here.

a curriculum writer may not know exactly what this means. I would add, 'i.e. the unit has "Is a Standalone Course" checked, or the unit belongs to a course.'

**Review by davidsbailey — APPROVED** [review]

**Comment by bethanyaconnor** [comment]
Thanks @davidsbailey! Still not completely sure what terms are eng-specific and which aren't. I'll update.

**Comment by davidsbailey** [comment]
> Thanks @davidsbailey! Still not completely sure what terms are eng-specific and which aren't. I'll update.

No worries! We do want "course version" to catch on :-) I still owe an illustration of where we are in the naming / renaming of things.

**Review by dmcavoy — APPROVED** [review]

**Pull request merged**